### PR TITLE
Removes gatfruit

### PR DESCRIPTION
**Pull request by Thunder12345**
## About The Pull Request

Removes gatfruit, its sprites, and all references to it.

## Why It's Good For The Game

Gatfruit is a completely balance ruining plant which adds nothing good to the game, and so should not be allowed to remain. As is the way with botany, once any quantity at all is available it rapidly multiplies into an effectively infinite amount. Barring a ground up revamp of how hydroponics works, this will always be the case.

 There are three scenarios which can occur when gatfruit comes in contact with the station in any way:

1. An antagonist obtains a supply of gatfruit. They now have an effectively infinite supply of revolvers, an exceptionally powerful weapon that can 2-shot an unarmoured human, and doesn't take much to kill even the best protected individuals. They can now murderbone with an infinite supply of ammo and burn through dozens of TCs worth of .357 ammunition at no ongoing cost.

2. A non-antagonist obtains a supply of gatfruit. As above they have infinite ammo, if they use this to validhunt this it an extremely unfun situation for any antagonists who find themselves faced with an endless stream of bullets.

3. A supply of gatfruit is made public. Everyone now has free revolvers, almost every other combat mechanic becomes obsolete because why would you use anything else. From here the round goes in only one direction, and that is coming to a screeching halt as the shuttle is called because of the inevitable mass violence.

TL;DR gatfruit has no good interactions with the station, serving only to overturn all sense of balance and turn the round into a bloodbath.

## Changelog
:cl:
del: Gatfruit has been removed
/:cl:

### CODE REVIEW

**Comment by InsaneRed** [comment]
so many ways to fix this issue without removing it

**Comment by interestingusernam3** [comment]
ided

**Comment by ExcessiveUseOfCobblestone** [comment]
I think limiting the spawns (maybe seed vault doesn’t need it) would be a preferred nerf for me than just outright removing it.

**Comment by itseasytosee** [comment]
Limiting spawns won't change the fact that the item is inherently unfun and unbalanced. This is not something you can fix by monkeying with stats, it needs a redesign.

**Comment by BebeYoshi** [comment]
I never understood how a plant that literally goes against all the plants standards (potency to make stronger, play around with reagents) was in game, the only true loss here is sulfur reagent gene, mind adding it to another plant?

**Comment by Thunder12345** [comment]
> 
> 
> so many ways to fix this issue without removing it
> 
> some ideas are : each seed just gives you 1 gun and gatfruit itself is bulky so you cant carry alot in your inventory

This just means you use multiple plant trays to grow it, and carry a backpack of revolvers at a time with a hidden stash of gatfruit in a crate somewhere.


> 
> 
> I think limiting the spawns (maybe seed vault doesn’t need it) would be a preferred nerf for me than just outright removing it.

Similarly, this just means you get it from another spawn and then go to the seed vault to mass produce it in peace.

Both of these suggestions are effectively bandaids for the real problem, which is obtaining a large stash of revolvers purely through off station content where you do not interact with the crew, and are never at any real risk of being caught while preparing.

**Comment by InsaneRed** [comment]
at best you get 2 seeds from the podpeople and you can make the gatfruit not give seeds upon putting it in the seed machine which gets you like 2 at most

**Comment by Burgermancoding** [comment]
make bulky

**Comment by Livrah** [comment]
Isn't this part of the upcoming lavaland loot rebalance?

**Comment by interestingusernam3** [comment]
>Isn't this part of the upcoming lavaland loot rebalance?

ETA: 5 Aeons

**Comment by CRITAWAKETS** [comment]
You could also just make it not make .357 revolvers but another weaker gun instead, like .38 revolvers.

**Comment by tf-4** [comment]
A decently significant unwrapping time would also be nice

**Comment by necromanceranne** [comment]
I know someone who wanted to add peashooters on another repo. Some silly joke gun that lets you shoot mostly harmless peas at people to fatten them up. Seemed like a neat replacement for gatfruit that is far more in line with crew side gear.

**Comment by necromanceranne** [comment]
I think I've mostly seen gatfruit used more for validhunting in deeply late rounds more than being used by lone antags. Most late round spawns just can't deal with multiple revolvers.

**Comment by AndroidMK2** [comment]
Just no

**Comment by ChungusGamer666** [comment]
Petty PR.
Gatfruit is exceedingly rare to be seen actually used. Hot take: It's fine for people to have OP gear late into the shift, or just very rarely due to random chance.

**Comment by Farquaar** [comment]
Why? Like Cobby said, it's incredibly rare that someone will find the seed vault, grow gatfruit, and bring it back stationside.

A rare seed that you only ever see used once in a blue moon doesn't really affect the balance of the game at all. If it was a problem that needed to be nerfed, there are more creative ways to reduce its power without removing one of the most unique rare plants in the game.

Not to mention, seed vault podpersons will lose their main non-explosive defence tool. Not as big a deal, but still worth noting.

**Comment by ATH1909** [comment]
you have not lived until you've felt the raw power of dual-wielding .357 revolvers, discarding them, and then pulling out two more from your belt like reaper from overwatch. it feels so fucking satisfying, man.

as critawakets said, a much better nerf would be to just replace the .357s with .38s, which don't two-shot crit people.

**Comment by wraith-54321** [comment]
there are much better options then just hitting the remove button, as someone said, just make them give much weaker guns

**Review by Mothblocks — CHANGES_REQUESTED** [review]
> I think limiting the spawns (maybe seed vault doesn’t need it) would be a preferred nerf for me than just outright removing it.

+1

**Comment by Rohesie** [comment]
I'd prefer the outright removal, but the other maintainers have spoken.
Is this a change you're willing to make?

**Comment by NoxVS** [comment]
While on the topic of nerfing the ability to obtain gatfruit, I don’t think seed vault is even the problem. A ruin that doesn’t have a guaranteed chance to spawn and then (assuming it hasn’t changed from how it used to be) doesn’t have a guaranteed chance to have gatfruit. It isn’t a reliable method of finding gatfruit. 

Xenobio, on the other hand, can reliably get gatfruit to start growing it every single round. Because gatfruit is a food before you break the revolver out of it, it can be spawned using silver extracts.

**Comment by itseasytosee** [comment]
I'd say its probably a better choice to rip the band-aid off and remove it rather than "balancing it" into useless obscurity like the magic 8-ball

**Comment by Mothblocks** [comment]
i think guns coming from plants is kinda funny

**Comment by MrMelbert** [comment]
For the record, all the places you can currently get gatfruit:
- Seed vault (miner, lavaland only)
- demonic portals (miner, icebox only)
- Cargo drones (cargo tech / qm)
- Xenobiology (scientist)

As a "botany player :tm: " I think gatfruit is a little overplayed and lame, since (like BebeYoshi says) it doesn't function how all other plants work but is just a source of gamer gear (potency doesn't matter, reagents don't matter end result isn't even a plant...)

as a plant man you can either: leverage your knowledge and skill of genes and traits to craft a neat mix of a death plant. orrr you can grow a gatfruit. nothing else about it, it's just a gun

alternatives: make it grow .38 instead of .357, make the number of ammo it starts with scale on potency, make the reagent contents affect the gun (somehow??)

**Comment by Mothblocks** [comment]
> alternatives: make it grow .38 instead of .357, make the number of ammo it starts with scale on potency (which never actually matters but is cool flavor sort of)

im fine with this

**Comment by Mothblocks** [comment]
not that i know what a .38 is but i like the idea of potency scaling to ammo

**Comment by Mothblocks** [comment]
would seem to me the problem is availability more than anything else though

**Comment by MrMelbert** [comment]
> 
> not that i know what a .38 is but i like the idea of potency scaling to ammo

.38 is the detective's revolver (which would about half the damage of guns from gatfruit)

**Comment by optimumtact** [comment]
this was only still in the game because kor liked it

Getting the gun should require you to grow a perfect 100% potency plant

otherwise it should spawn water guns and other less useful guns as a joke

**Comment by Mothblocks** [comment]
thats also funny i like that

**Comment by EricZilla** [comment]
Would be nice if the revolver that came out couldnt be reloaded, and maybe actually looked like a plant?

**Comment by TheFakeElon** [comment]
esports ready tg

**Comment by LemonInTheDark** [comment]
> 
> 
> thats also funny i like that

**Comment by Mothblocks** [comment]
we don't want the pr in its current state and i want to clean up the tracker
